### ISSUE TITLE
Error Getting Team ID

### ISSUE THREAD

**Issue by belzedaar** [ljmerza/py-our-groceries#13]
I think something has changed in the OurGrocieries web page this talks to because this no longer finds the team ID:
```
 File "/var/lib/snips/skills/snips-skill-our-groceries/venv/lib/python3.5/site-packages/ourgroceries/__init__.py", line 80, in login
    await self._get_team_id()
  File "/var/lib/snips/skills/snips-skill-our-groceries/venv/lib/python3.5/site-packages/ourgroceries/__init__.py", line 110, in _get_team_id
    self._category_id = re.findall(REGEX_CATEGORY_ID, responseText)[0]
IndexError: list index out of range
```

I haven't changed username/password and the same login still works on the website. I'll see if I can debug it on my end, but I thought I'd report it as well.

**Comment by belzedaar** [comment]
Seems like the exception thrown is because g_categoryListId is no longer in the page.

**Comment by andreasbrett** [comment]
I can confirm that in the source code of the `/your-lists/` subpage there is no `g_categoryListId` anymore which causes the code to crash (as there's no error handling in the regex extraction). The `g_teamId` can still be extracted but the `g_categoryListId` is gone. It can be extracted from `g_staticMetalist` though as there all shopping lists + the master list + the category list are defined (see below).

```
<script type="text/javascript">
   var g_teamId = "***************";
   var g_listId = "";
   var g_fromShoppingListId = null;
   var g_addUncategorizedItem = "LEAVE";
   var g_capitalizeNewItems = "NONE";
   const g_crossedOffOrder = "RECENT_BOTTOM";
   const g_masterListOrder = "RECENT_TOP";
   const g_emptyListsLast = false;
   const g_staticMetalist = [{"name":"Rewe","id":"***","notes":"","notesHtml":"","listType":"SHOPPING","versionId":"***","items":null},{"name":"","id":"***","notes":"","notesHtml":"","listType":"MASTER","versionId":"***","items":null},{"name":"","id":"***","notes":"","notesHtml":"","listType":"CATEGORY","versionId":"***","items":null}]
   const g_listSchemaVersion = 1;
</script>
```

The following should do the trick: replacing https://github.com/ljmerza/py-our-groceries/blob/f3bfcf6b7a4c8a4e2757cd3446db8807fc1147bb/ourgroceries/__init__.py#L110

with

```
static_metalist = json.loads(re.findall(REGEX_STATIC_METALIST, responseText)[0])
categoryList = [list for list in static_metalist if list['listType'] == 'CATEGORY'][0]
self._category_id = categoryList['id']
```

and additionally
- importing json via `import json`
- defining `REGEX_STATIC_METALIST = r'g_staticMetalist = (\[.*\]);'`

**Comment by ljmerza** [comment]
merged to v1.3.9
